cms/src/test/java/uk/nhs/digital/externalstorage/s3/TestS3ObjectDescriber.java

```diff
@@ -0,0 +1,35 @@
+package uk.nhs.digital.externalstorage.s3;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import org.junit.Test;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class TestS3ObjectDescriber {
```

Comment:
How about calling it `St3NodeDescriberTest` so that the name of the class doesn't sound like a command but is closeer to a noun, instead.